tools/rpm2img

```diff
@@ -383,16 +389,23 @@ cp /host/tools/bootconfig/empty-bootconfig.data "${PRIVATE_MOUNT}/bootconfig.dat
 mkfs.ext4 -b 4096 -i 4096 -I 256 -d "${PRIVATE_MOUNT}" "${PRIVATE_IMAGE}" "${partsize[PRIVATE]}M"
 dd if="${PRIVATE_IMAGE}" of="${OS_IMAGE}" conv=notrunc bs=1M seek="${partoff[PRIVATE]}"
 
-# BOTTLEROCKET-DATA
+# BOTTLEROCKET-DATA and BOTTLEROCKET-DATA-ALT
 
 # If we build on a host with SELinux enabled, we could end up with labels that
 # do not match our policy. Since we allow replacing the data volume at runtime,
 # we can't count on these labels being correct in any case, and it's better to
 # remove them all.
 UNLABELED=$(find "${DATA_MOUNT}" \
     | awk -v root="${DATA_MOUNT}" '{gsub(root"/","/"); gsub(root,"/"); print "ea_rm", $1, "security.selinux"}')
+UNLABELED_ALT=$(find "${DATA_ALT_MOUNT}" \
+    | awk -v root="${DATA_ALT_MOUNT}" '{gsub(root"/","/"); gsub(root,"/"); print "ea_rm", $1, "security.selinux"}')
 mkfs.ext4 -d "${DATA_MOUNT}" "${BOTTLEROCKET_DATA}" "${partsize[DATA]}M"
+mkfs.ext4 -d "${DATA_ALT_MOUNT}" "${BOTTLEROCKET_DATA_ALT}" "${partsize[DATA-ALT]}M"
 echo "${UNLABELED}" | debugfs -w -f - "${BOTTLEROCKET_DATA}"
+echo "${UNLABELED_ALT}" | debugfs -w -f - "${BOTTLEROCKET_DATA_ALT}"
+
+dd if="${BOTTLEROCKET_DATA_ALT}" of="${OS_IMAGE}" conv=notrunc bs=1M seek="${partoff[DATA-ALT]}"
```

Comment:
It might be better to just write the same filesystem image to `DATA-A` and `DATA-B` if it exists?

Reply:
Can I do that even if the filesystem sizes are different?